plenum/common/ledger_manager.py

```diff
@@ -116,6 +128,10 @@ def addLedger(self, iD: int, ledger: Ledger,
             verifier=MerkleVerifier(ledger.hasher)
         )
 
+        if iD == POOL_LEDGER_ID:
+            self._client_seeder.add_ledger(iD, ledger)
```

Comment:
Although libindy catches up pool ledger only, I think some clients may want to catchup other ledgers. Do we really need to restrict client service to work with Pool ledger only?

Reply:
No real need to restrict client service, it was just logic of original code which I kept

Reply:
Removed restriction